src/browser/BrowserAction.cpp

```diff
@@ -406,11 +406,11 @@ QString BrowserAction::getErrorMessage(const int errorCode) const
     case ERROR_KEEPASS_CANNOT_DECRYPT_MESSAGE:
         return QObject::tr("Cannot decrypt message");
     case ERROR_KEEPASS_TIMEOUT_OR_NOT_CONNECTED:
-        return QObject::tr("Timeout or cannot connect to KeePassXC");
+        return QObject::tr("Cannot connect to KeePassXC. Check that browser integration is enabled in KeePassXC settings.");
     case ERROR_KEEPASS_ACTION_CANCELLED_OR_DENIED:
         return QObject::tr("Action cancelled or denied");
     case ERROR_KEEPASS_CANNOT_ENCRYPT_MESSAGE:
-        return QObject::tr("Cannot encrypt message or public key not found. Is Native Messaging enabled in KeePassXC?");
+        return QObject::tr("Message encryption failed. Is KeePassXC running?");
```

Comment:
Hmmm, I was confused when I first reviewed this PR. Of course KeePassXC is running or you wouldn't get this error message in the first place. Perhaps just leave it at "Message encryption failed."

Reply:
This error code is not used anywhere in the code.